### DISCUSSION TITLE
MacOS Audio Hardware Disappears

### DISCUSSION THREAD

**Discussion by gene96817** [jamulussoftware/jamulus#1833] [Bug Investigation]
Has anyone else experienced the audio hardware disappearing?
Under About This Mac/System Report/Hareware/Audio, nothing is seen.
Under System Preferences/Sound the internal input and outputs are not shown.
Everything appears normal right after a restart. Shortly later, the audio hardware disappears.

I thought it was a hardware failure on my MacBook Pro. However, I transferred my all my software to a MacBook Air with Migration Assistant and now I see this same disappearing audio hardware on the second machine. Since I originally perceived this as a hardware problem that slowly got worst over the past few weeks, I don't have a clear idea when the first symptom appeared. It might be related to MacOS 11.x. It happens with Jamulus 3.7 or 3.8 installed. 

Any suggestions of what software suspects to investigate?

**Comment by ann0see** [comment]
I‘d ask the Apple support unless it’s clearly not present if Jamulus is not installed.

**Reply by ann0see** [reply]
Maybe you can look if `lsusb` (if that works on macOS) still shows the hardware if it it disappears. I assume there’s also a way to find information in the "System Info" App. But I don’t think I can help you much with that unfortunately.

**Comment by gene96817** [comment]
Thanks for the comments.... I am in the hunt for some code that is incompatible with macOS 11.4. I seem to have accumulated an awful lot of apps. It will take a while.

**Comment by gene96817** [comment]
Thank you all for the replies and suggestions. It turns out, one of the applets in my host-based intrusion protection system (HIPS) monitors the mic, speaker, and camera as part of protecting privacy. There was an incompatibility with macOS Big Sur. The HIPS has worked so well for so many years that it was under my radar.
